local-rest-scorer/src/main/java/ai/h2o/mojos/deploy/local/rest/controller/ModelsApiController.java

```diff
@@ -116,4 +118,26 @@ public ModelsApiController(MojoScorer scorer, SampleRequestBuilder sampleRequest
   public ResponseEntity<ScoreRequest> getSampleRequest() {
     return ResponseEntity.ok(sampleRequestBuilder.build(scorer.getPipeline().getInputMeta()));
   }
+
+  private List<CapabilityType> setSupportedCapabilities() {
+    List<CapabilityType> capabilityTypes = new ArrayList<>();
+    capabilityTypes.add(CapabilityType.SCORE);
+    ShapleyLoadOption enabledShapleyTypes = scorer.getEnabledShapleyTypes();
+    switch (enabledShapleyTypes) {
+      case ALL:
+        capabilityTypes.add(CapabilityType.CONTRIBUTION_ORIGINAL);
+        capabilityTypes.add(CapabilityType.CONTRIBUTION_TRANSFORMED);
+        break;
+      case ORIGINAL:
+        capabilityTypes.add(CapabilityType.CONTRIBUTION_ORIGINAL);
+        break;
+      case TRANSFORMED:
+        capabilityTypes.add(CapabilityType.CONTRIBUTION_TRANSFORMED);
+        break;
+      case NONE:
+      default:
+        break;
+    }
+    return capabilityTypes;
```

Comment:
100% fine and no issues, just providing the below to share style knowledge and some common tips:

- (naming convention) Since method returns and doesn't set member field, consider renaming method from `setSup...` to something like `assembleSupp..` or some other synonym.
- (reducing reliance/risk on specific instance initialization-order) Can reduce reliance on initialized instance variable by passing in `value of scorer.getEnabl..()` instead of looking it up from inside the method.  Would also free method up to become static.
- (readability) Returning immediately, instead of breaking, often allows reader to not finish reading method or keep state in their head as they scroll.  Similarly, can have each `case` return the full list outright without building (e.g., `List.of(..., ..., ....)`.  Means a few extra words of code, but is explicit, immutable, and no building logic involved.